### PR TITLE
Level work

### PR DESCRIPTION
**Pull request by zerox1990**

### CODE REVIEW

File: SGoopas/ProjectSettings/ProjectVersion.txt [hunk 1]
```diff
@@ -1 +1 @@
-m_EditorVersion: 2017.3.0f3
```
Comment (dannyflax) [inline SGoopas/ProjectSettings/ProjectVersion.txt]: don’t commit this change please

**Comment by dannyflax** [comment]
Also if you could give a more detailed description of the changes + maybe include a screenshot of your level in this PR, that would be superb

File: SGoopas/ProjectSettings/ProjectVersion.txt (hunk 1 above)
Comment (zerox1990) [reply SGoopas/ProjectSettings/ProjectVersion.txt]: Twice?

Comment (dannyflax) [reply SGoopas/ProjectSettings/ProjectVersion.txt]: What? This comment is about ProjectVersion.txt. The twice thing is about the .Unity files

**Comment by zerox1990** [comment]
There are two day time unity scenes because on is the level inside the assets folder and one is the demo that came with the urban asset pack and is in the urban assets folder.

**Pull request merged**